lib/features/support/capabilities/capabilities.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
```

Comment:
Again, this largely resulted from factoring code out of `app_automate_driver.rb`, but with more additions.